.sdk/FINSDK.cpp

```
@@ -0,0 +1,235 @@
+        super.onDestroy();
+        finWrapper.stopKnowledgeBlock();
+    }
+}
+#endif
+
+---
+
+### **CMakeLists.txt**
+
+```cmake
+cmake_minimum_required(VERSION 3.10.2)
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If VERSION is a macro then please configure it.

(unknownMacro)